docs/api/errors.rst

```diff
@@ -39,6 +39,31 @@ Errors are meant to be managed at the project level.
 However, a basic ref:`<set of errors codes <errors>`_ is provided in *Cliquet*.
 
 
+Precondition errors
+===================
+
+When an operation is cancelled because of concurrency (using ETags), a
+``412 Precondition Failed`` error response is returned.
```

Comment:
That's rather technical; maybe we could vulgarize a little with a sample use case?